### DISCUSSION TITLE
Gentagende login problem

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#25] [Bugs] [answered]
Når logger ind fra portalen, og trykker mail eller adgangskode forkert, så godtager den ikke den rigtige mail/adgangskode. Man skal først refreshe browseren, før en ny mail/adgangskode godtages.

**Comment by henrikbak** [comment]
@MartinQwist Jeg kan ikke genskabe problemet i portalen. 

Jeg har prøvet med min rigtige email og en forkert adgangskode, for derefter at logge ind med min rigtige email og adgangskode. Jeg har også prøvet med en forkert email, som naturligvis ikke virker, for derefter at logge ind med min rigtige email og adgangskode. Alt sammen uden at refreshe siden i browseren.

Kan du på nogen måde beskrive den fejl i oplever i lidt flere detaljer?

**Comment by MartinQwist** [comment]
Vi kan genskabe problemet på Safari på Mac. De fleste af vores arrangører bruger mac.

**Comment by mikkelsl** [comment, marked as answer]
Jeg har lavet en opgave på det i Development: https://github.com/orgs/xooom-community/projects/1/views/1?pane=issue&itemId=69511733
